### ISSUE TITLE
Please add support for .svg Images

### ISSUE THREAD

**Issue by Niedes** [collective/collective.autoscaling#3]
We have errors, when we want to upload .svg images with collective.autoscaling enabled like.
` ERROR plone.transformchain Unexpected error whilst trying to apply transform chain
 ...
 Forbidden: Form authenticator is invalid.`
We have to patch around with 
`if '.svg' in imageField.filename:
            continue`
in `resizing.py` like

```
def scale_images(obj, request):
    imageFieldsNames = get_image_fields(obj)
    if not imageFieldsNames:
        return 0

    resized = 0
    for imageFieldName in imageFieldsNames:
        imageField = getattr(obj, imageFieldName)
        if '.svg' in imageField.filename:
            continue

        original_file = StringIO(imageField.data)
        ...
```

to make it work.

**Comment by hvelarde** [comment]
please provide a pull request with your patch.

**Comment by laulaz** [comment]
@Niedes good news (even if a bit late), this is now fixed, thanks to the work of @marclava in c6eda0730644b775398e66124805113cb6203037

**Comment by laulaz** [comment]
For the record, this is the debug message you get when uploading an SVG :
```
autoscaling <plone.namedfile.file.NamedBlobImage object at 0x1143655f8> : cannot identify image file <_io.BytesIO object at 0x1143a4d00>
```
